### ISSUE TITLE
Add hover functionality to the 3D Viewports

### ISSUE THREAD

**Issue by Reousa** [moonwards1/Moonwards-Virtual-Moon#516]

**Comment by Zach777** [comment]
Like hover as in the mouse hovers over a button and the button reacts because of the hover? SlidePresentation already does that.
 First person mode does not allow hovering though.

**Comment by Reousa** [comment]
Yes, I'll still be taking a thorough look at this one, nothing urgent at the moment.
